themes-default/slim/views/history.mako

```diff
@@ -22,17 +22,34 @@ const startVue = () => {
         metaInfo: {
             title: 'History'
         },
+        store,
         data() {
             return {
-                header: 'History'
+                header: 'History',
+                limit: parseInt('${limit}', 10)
             };
         },
+        computed: Object.assign({
+            layout: {
+                get() {
+                    const { config } = this;
+                    return config.layout.history;
+                },
+                set(layout) {
+                    const { $store } = this;
+                    const page = 'history';
+                    $store.dispatch('setLayout', { page, layout });
+                }
+            }
+        }),
         mounted() {
+            const { layout, config } = this;
```

Comment:
I didn't checkout the branch, but...
[We already had an issue with this on the schedule page.](https://github.com/pymedusa/Medusa/commit/7cd3d70a682ef13e9001c2ed2c530cf13856b0ef)

Reply:
This uses `setLayout` which currently reloads the page so it should be fine.

Reply:
It's not the same issue. The issue was they weren't available **when the page loaded**.

Reply:
Since it's injected to the root instance it should be available in the store before this component is even loaded as mixins run their mounted, etc. before the component itself.

You should actually be able to revert the change you needed for schedule if you're using this branch.

Reply:
I just checked and this *is* an issue (on history page, with subtitles enabled),
unless something else is wrong.

Reply:
If you add `console.log(config.subtitles.enabled)` under this what do you get?

Reply:
I put the log below this line
And another log above https://github.com/pymedusa/Medusa/blob/990d701805c2ea959678c47093971afe47ce26f2/themes-default/slim/views/history.mako#L67

`null`. Both locations.

Reply:
What if you use `this.config` in both places?

Reply:
It's the same.
The issue is when mounted is called, the config has yet to be committed to the store.
Vue doesn't wait for the global mixin's `created()` to finish getting the config.

Reply:
What do you suggest we switch this to then?

Reply:
Actually no, I keep getting different errors. But I already messed with some files so please hold.

Reply:
Actually no, the one before `if (config.subtitles.enabled) { ` line
can't use `this` because it's jQuery, I believe.

Reply:
@OmgImAlexis 
I don't know if there's anything we can do about it right now.
Only solution I have in mind is the one I used on `schedule.mako`, which works on this page too.